experimental/framework/frs-client/src/FrsClient.ts

```diff
@@ -0,0 +1,176 @@
+        container: Container,
+    ): Promise<[FluidContainer, FrsContainerServices]> {
+        const rootDataObject = await requestFluidObject<RootDataObject>(container, "/");
+        const fluidContainer = new FluidContainer(container, rootDataObject);
+        const containerServices = this.getContainerServices(container);
+        return [fluidContainer, containerServices];
+    }
+
+    private getContainerServices(
+        container: Container,
+    ): FrsContainerServices {
+        return {
+            audience: new FrsAudience(container),
+        };
+    }
+
+    private async getContainerCore(
+        containerConfig: FrsContainerConfig,
+        containerSchema: ContainerSchema,
+        createNew: boolean,
+    ): Promise<[FluidContainer, FrsContainerServices]> {
+        const runtimeFactory = new DOProviderContainerRuntimeFactory(
+            containerSchema,
+        );
+        const module = { fluidExport: runtimeFactory };
+        const codeLoader = { load: async () => module };
+        const urlResolver = new FrsSimpleUrlResolver(this.connectionConfig, containerConfig.id);
+        const loader = new Loader({
+            urlResolver,
+            documentServiceFactory: this.documentServiceFactory,
+            codeLoader,
+            logger: containerConfig.logger,
+        });
+
+        let container: Container;
+
+        if (createNew) {
+            // We're not actually using the code proposal (our code loader always loads the same module
+            // regardless of the proposal), but the Container will only give us a NullRuntime if there's
+            // no proposal.  So we'll use a fake proposal.
+            container = await loader.createDetachedContainer({
+                package: "no-dynamic-package",
+                config: {},
+            });
+            await container.attach({ url: containerConfig.id });
+        } else {
+            // Request must be appropriate and parseable by resolver.
+            container = await loader.resolve({ url: containerConfig.id });
+            // If we didn't create the container properly, then it won't function correctly.
+            // So we'll throw if we got a
+            // new container here, where we expect this to be loading an existing container.
+            if (container.existing === undefined) {
+                throw new Error("Attempted to load a non-existing container");
+            }
+        }
+        return this.getFluidContainerAndServices(container);
+    }
+}
+
+/**
+ * FrsClient static class with singular global instance that lets the developer define
+ * all Container interactions with the Frs service
+ */
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export class FrsClient {
+    private static globalInstance: FrsClientInstance | undefined;
+
+    static init(connectionConfig: FrsConnectionConfig) {
+        if (FrsClient.globalInstance) {
+            console.log(
```

Comment:
This log statement is inaccurate (you're creating a new instance in all cases).  If you want to reuse, then missing a return here maybe?  However it seems like using an `FrsClient` with config different from the one you tried to specify might be pretty disastrous (I'm uploading my document to an unexpected location?) so reuse might not be a good idea.  Perhaps we should consider dropping the global instance.

Also remove log before checkin.

Reply:
Yes, it was missing the return. The client shouldn't be initiated again. I've kept the global instance for now as it makes for easy portability when consuming the client. Otherwise, you need to store it in some context or pass it around as props/parameters anywhere you need to use it. The `FrsClientInstance` class that it uses to instantiate the global instance is also exported here, so it should allow for easy testability of the underlying logic.

Also, I've removed the log statement and replaced it with `debug` since we do not have access to the logger here